testsuite/shared/src/main/java/org/jboss/as/test/shared/observability/setuptasks/AbstractSetupTask.java

```diff
@@ -16,29 +16,33 @@
 import org.jboss.dmr.ModelNode;
 
 public abstract class AbstractSetupTask implements ServerSetupTask {
-    protected ModelNode clearAttribute(String address, String attributeName) {
+    public static ModelNode clearAttribute(String address, String attributeName) {
         ModelNode op = Operations.createOperation("write-attribute", Operations.createAddress(SUBSYSTEM, address));
         op.get("name").set(attributeName);
         return op;
     }
 
-    protected ModelNode writeAttribute(String subsystem, String name, String value) {
-        return Operations.createWriteAttributeOperation(Operations.createAddress(SUBSYSTEM, subsystem), name, value);
+    public static ModelNode writeAttribute(String subsystem, String name, String value) {
+        return writeAttribute(Operations.createAddress(SUBSYSTEM, subsystem), name, value);
     }
 
-    protected void executeOp(final ManagementClient client, final ModelNode op) throws IOException {
+    public static ModelNode writeAttribute(ModelNode address, String name, String value) {
+        return Operations.createWriteAttributeOperation(address, name, value);
+    }
+
+    public static void executeOp(final ManagementClient client, final ModelNode op) throws IOException {
         executeOp(client.getControllerClient(), Operation.Factory.create(op));
     }
 
-    protected void executeOp(final ModelControllerClient client, final Operation op) throws IOException {
+    public static void executeOp(final ModelControllerClient client, final Operation op) throws IOException {
         final ModelNode result = client.execute(op);
         if (!Operations.isSuccessfulOutcome(result)) {
             throw new RuntimeException("Failed to execute operation: " + Operations.getFailureDescription(result)
                     .asString());
         }
     }
 
-    protected ModelNode executeRead(final ManagementClient managementClient, ModelNode address) throws IOException {
+    public static ModelNode executeRead(final ManagementClient managementClient, ModelNode address) throws IOException {
         return managementClient.getControllerClient().execute(Operations.createReadResourceOperation(address));
     }
```

Comment:
Is there a reason these were made public and static?

Reply:
I'm not sure, tbh. I removed static from those and fixed up the one compile problem it caused.